### PR TITLE
Sample app

### PR DESCRIPTION
**Pull request by imtoori**
Moved sample app to separate repo
Created new sample app with tutorial steps

### CODE REVIEW

File: packages/stream_chat_flutter/example/pubspec.yaml [hunk 1]
```diff
@@ -13,30 +27,53 @@ dependencies:
     path: ../
   stream_chat_persistence:
     path: ../../stream_chat_persistence
```
Comment (Nash0x7E2) [inline packages/stream_chat_flutter/example/pubspec.yaml:29]: These relative paths should be removed.

**Review by Nash0x7E2 — APPROVED** [review]
LGTM

File: packages/stream_chat_flutter/example/pubspec.yaml (hunk 1 above)
Comment (imtoori) [reply packages/stream_chat_flutter/example/pubspec.yaml:29]: Are you sure?
I probably agree on the persistence package, but why should we remove the relative import of the sdk?

Comment (Nash0x7E2) [reply packages/stream_chat_flutter/example/pubspec.yaml:29]: Flutter is usually installed at `sdk: flutter`, not `path: ../`. A user may not have the framework installed in that directory.

Comment (imtoori) [reply packages/stream_chat_flutter/example/pubspec.yaml:29]: expand the diff :)

**Pull request merged**